platform-sdk/swirlds-base/src/main/java/com/swirlds/base/utility/ExceptionUtils.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.base.utility;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Objects;
+
+public enum ExceptionUtils {
```

Comment:
This is only used in the browser UI. Please move it to swirlds-common or if possible platform-gui or platform-core.